README.md

```diff
@@ -20,6 +20,7 @@ This repository serves as a space for the Ansible Community Working Group to man
 * Ansible development: [ansible/ansible](https://github.com/ansible/ansible)
 * Working Groups
   - [Community Working Group](https://github.com/ansible/community/tree/master/group-community)
+  - [Amazon Web Services Working Group](https://github.com/ansible/community/tree/master/group-aws)
```

Comment:
I would sort them alphabetically.
I also wonder why not just call it AWS Working Group as the target audience exactly knows what AWS is (and even only may know it under the acronym name ;-)).